### PR TITLE
Fix indentation of metadata in ansible-doc

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
This fix adds indentation fix for metadata tag.
Also, adds support block which was skipped due to
metadata_version check.

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - Bugfix Pull Request
 - Docs Pull Request

##### COMPONENT NAME
lib/ansible/cli/doc.py

##### ANSIBLE VERSION

```
2.7devel
```

Before patch - 

screen shot 2018-06-05 at 10 05 23 am
screen shot 2018-06-05 at 10 05 14 am

After patch - 

screen shot 2018-06-05 at 10 04 30 am
screen shot 2018-06-05 at 10 04 20 am

### CODE REVIEW

File: lib/ansible/cli/doc.py [hunk 1]
```diff
@@ -508,6 +500,11 @@ def get_man_text(self, doc):
                 text.append('%s: %s' % (k.upper(), textwrap.fill(CLI.tty_ify(doc[k]), limit - (len(k) + 2), subsequent_indent=opt_indent)))
             elif isinstance(doc[k], (list, tuple)):
                 text.append('%s: %s' % (k.upper(), ', '.join(doc[k])))
+            elif k == 'metadata':
+                metadata_block = self.get_metadata_block(doc)
```
Comment (mkrizek) [inline lib/ansible/cli/doc.py:504]: Does this throw any exception? In the code you're removing it's inside `try:`.

Comment (Akasurde) [reply lib/ansible/cli/doc.py:504]: No. I think this is necessary with new code.

**Review by mkrizek — APPROVED** [review]

**Comment by maxamillion** [comment]
shipit

File: lib/ansible/cli/doc.py [hunk 2]
```diff
@@ -428,26 +428,18 @@ def get_support_block(doc):
...
-        if doc['metadata'].get('metadata_version') in ('1.0', '1.1'):
-            return ["  * This module is maintained by %s" % support_level_msg[doc['metadata']['supported_by']]]
-
-        return []
+        return ["  * This module is maintained by %s\n" % support_level_msg[doc['metadata']['supported_by']]]
 
     @staticmethod
     def get_metadata_block(doc):
-        text = []
-        if doc['metadata'].get('metadata_version') in ('1.0', '1.1'):
```
Comment (bcoca) [inline lib/ansible/cli/doc.py:439]: won't omission of this 'if'  break those w/o metadata entry?

**Pull request closed without merging**